tests/integration/linux_bridge_test.py

```diff
@@ -375,16 +376,35 @@ def test_activate_empty_bridge_does_not_blocked_by_dhcp():
         assertlib.assert_state(desired_state)
 
 
+def test_port_vlan_not_implemented(port0_up):
+    bridge_name = TEST_BRIDGE0
+    port_name = port0_up[Interface.KEY][0][Interface.NAME]
+    port_vlan_config = {
+        LinuxBridge.Port.VLAN_SUBTREE: {
+            LinuxBridge.Port.Vlan.TYPE: LinuxBridge.Port.Vlan.ACCESS_TYPE,
+            LinuxBridge.Port.Vlan.TAG: 101,
+        }
+    }
+
+    bridge_state = _create_bridge_subtree_config(
```

Comment:
This should probably be named `port_state`.

Reply:
It looks correct to me. The returned state is of the bridge subtree which includes the ports and other parameters. 
The new argument confused things.